keps/sig-api-machinery/2876-crd-validation-expression-language/kep.yaml

```diff
@@ -22,17 +22,18 @@ see-also:
   - "/keps/sig-api-machinery/95-custom-resource-definitions"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
```

Comment:
"Scalability evaluation. Evaluate CRD scalability with validation rules against previous scale target." is listed in our GA criteria. Include our results in the KEP somewhere?

Reply:
Added. Thank you